src/openrct2/CMakeLists.txt

```diff
@@ -106,6 +106,13 @@ if (NOT DISABLE_NETWORK)
     endif ()
 endif ()
 
+if (NOT APPLE AND NOT MINGW)
+    # This is ugly hack to work around https://bugs.launchpad.net/ubuntu/+source/gcc-5/+bug/1568899.
+    # Once C++17 is enabled (and thus old compilers are no longer supported, this needs to be gone.
+    # We cannot simply detect the _compiler_ version, as the bug exists with the C++ _library_
+    target_link_libraries(${PROJECT} gcc_s gcc)
```

Comment:
GCC's turn to have a bug now :)

Reply:
Yes, GCC 5, first released 2015-04-22 and already fixed in newer versions.